src/gl/metal/vcShader.mm

```diff
@@ -155,7 +156,7 @@ bool vcShader_CreateFromTextInternal(vcShader **ppShader, const char *pVertexSha
         break;
     }
 
-    [_renderer.pipelines addObject:[_device newRenderPipelineStateWithDescriptor:pDesc error:&err]];
+    pShader->pipelines[i] = [_device newRenderPipelineStateWithDescriptor:pDesc error:&err];
```

Comment:
To confirm; this is still correct when line 122 looks like it does the same thing (at least when `i==0`)?

Reply:
It's not quite the same thing - line 122 also generates the reflection information for the constant buffers. Also, I'm not entirely sure we need a unique pipeline for each blend mode, but that's how it worked before. Finally, since this also caught me out, this loop starts at `i==1` not `i==0` - insanely easy to gloss over.

Reply:
Sorry; yes you're right. I totally missed that in the loop (I guess that's a negative for not magic numbers)